### PR TITLE
Add a note to the book

### PR DESCRIPTION
**Pull request by mashomee**
#1329

### CODE REVIEW

File: book/src/composite_templates.md [hunk 1]
```diff
@@ -167,6 +167,7 @@ Filename: <a class=file-link href="https://github.com/gtk-rs/gtk4-rs/blob/master
 ```
 
 Then we have to bind the template callbacks with [`bind_template_callbacks`](../docs/gtk4/subclass/widget/trait.CompositeTemplateCallbacksClass.html#tymethod.bind_template_callbacks).
+We also need to remove the `button` clicked callback implemented in `window/imp.rs` for this callback to work.
```
Comment (Hofer-Julian) [inline book/src/composite_templates.md:170]:
```suggestion
We also need to remove the `button.connect_clicked` callback implemented in `window/imp.rs`.
```

**Review by Hofer-Julian — COMMENTED** [review]
Looks good otherwise

**Comment by mashomee** [comment]
Shoud I Open another pull request, or this one just ok?

**Review by Hofer-Julian — APPROVED** [review]
Thanks!

This one is fine, I will merge it now :)

**Pull request merged**